src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,22 @@
-import React from 'react';
+import MovieCard from '../MovieCard';
+import { Movie } from '../../types/movie';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" data-cy="card" key={movie.imdbId}>
+        <MovieCard
+          title={movie.title}
+          description={movie.description}
+          imgUrl={movie.imgUrl}
+          imdbUrl={movie.imdbUrl}
+        />
```

Comment:
```suggestion
        <MovieCard {...movie} />
```
Here you can destructurize object if you want.